### PR TITLE
v3.0.1

### PR DESCRIPTION
**Pull request by whymarrh**
This PR represents v3.0.1, which will include #277.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [3.0.1] - 2020-09-15
```
Comment (Gudahtt) [inline CHANGELOG.md:9]: The links at the bottom of the changelog still need to be updated with this release

Comment (whymarrh) [reply CHANGELOG.md:9]: Good catch, thanks! 😅

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Comment by whymarrh** [comment]
> A new version of the package `@metamask/controllers` (3.0.1) was published at 2020-09-15T20:38:28.445Z from
[redacted]. The shasum of this package was `eea137df6c80c2590a75a057dab7d663ff462588`.

**Pull request merged**